moped-editor/src/utils/mapDrawHelpers.js

```diff
@@ -37,27 +36,91 @@ export const MODES = [
 
 const STROKE_COLOR = theme.palette.primary.main;
 const FILL_COLOR = theme.palette.primary.main;
-const CIRCLE_RADIUS = 20;
 
 const SELECTED_STYLE = {
   stroke: STROKE_COLOR,
-  strokeWidth: 2,
+  strokeWidth: 8,
   fill: FILL_COLOR,
-  fillOpacity: mapStyles.statusOpacities.selected,
+  fillOpacity: 0,
 };
 
 const HOVERED_STYLE = {
   stroke: STROKE_COLOR,
-  strokeWidth: 2,
+  strokeWidth: 8,
   fill: FILL_COLOR,
-  fillOpacity: mapStyles.statusOpacities.hovered,
+  fillOpacity: 0,
 };
 
 const DEFAULT_STYLE = {
   stroke: theme.palette.primary.main,
-  strokeWidth: 2,
-  fill: FILL_COLOR,
-  fillOpacity: mapStyles.statusOpacities.unselected,
+  strokeWidth: 4,
+  fill: theme.palette.secondary.main,
+  fillOpacity: 1,
+};
+
+/**
+ * Interpolate a feature width based on the zoom level of map
+ * Adapted from Mapbox GL JS linear interpolation using a formula linked below
+ * https://github.com/mapbox/mapbox-gl-js/blob/d66ff288e7ab2e917e9e676bee942dd6a46171e7/src/style-spec/expression/definitions/interpolate.js
+ * https://matthew-brett.github.io/teaching/linear_interpolation.html
+ * @param {number} currentZoom - Current zoom level from the map
+ * @param {number} minZoom - Minimum zoom level from the current bracket
+ * @param {number} maxZoom - Maximum zoom level from the current bracket
+ * @param {number} minPixelWidth - Minimum pixel width from the current bracket
+ * @param {number} maxPixelWidth - Maximum pixel width from the current bracket
+ * @return {number} Interpolated pixel width
+ */
+function linearInterpolation(
+  currentZoom,
+  minZoom,
+  maxZoom,
+  minPixelWidth,
+  maxPixelWidth
+) {
+  return (
+    ((currentZoom - minZoom) * (maxPixelWidth - minPixelWidth)) /
+      (maxZoom - minZoom) +
+    minPixelWidth
+  );
+}
```

Comment:
Check this out @johnclary, mike is out there doing science 💯 

https://matthew-brett.github.io/teaching/linear_interpolation.html

Reply:
🔥

Reply:
🧠👨‍🔬!